asabiris/orchestration/sendmsteams.py

```diff
@@ -45,14 +45,13 @@ async def send_to_msteams(self, msg):
 			L.warning("Invalid notification format: {}".format(e))
 			return
 
-
 		body = msg['body']
-		if not body['template'].startswith("/Templates/MSTeams/"):
-			raise PathError(use_case='MSTeams', invalid_path=body['template'])
+		template = body["template"]
 
-		body["params"] = body.get("params", {})
-		output = await self.JinjaService.format(body["template"], body["params"])
+		if not template.startswith("/Templates/MSTeams/"):
+			raise PathError(use_case='MSTeams', invalid_path=template)
```

Comment:
I guess these errors shoudl be caucht specifically, not just using except Exception

Reply:
Not sure what do you mean,